components/rich_text_editor/rich_text_editor.vue

```diff
@@ -114,6 +117,15 @@ export default {
       type: [Boolean, Object],
       default: false,
     },
+
+    /**
+     * suggestion object containing the items query function. When null, it does not add the plugin.
+     */
+    mentionSuggestion: {
+      type: Object,
+      default: null,
+      // required: true,
+    },
```

Comment:
Hmm is this an object type? Appears to be a function that returns an array of objects?

Reply:
changed the description to explain the object. Also introduced doc changes to make it easier to use this.

Reply:
its an object that contains the function, i did it this way so it can be further customized by product if needed. for example if they want to override the render function or the trigger character, all can be done in this object. I have documented it here:
https://github.com/dialpad/dialtone-vue/pull/1248/files#diff-8e1296ebf37f15b6140a0f899b263bf8b5461c291a8a33f3bb32f09006ed6534R71